pmlc/dialect/pxa/transforms/stencil_gemm.cc

```diff
@@ -293,6 +293,7 @@ class StencilGEMM : public StencilBase {
                                            tileAttr, mapOperands);
 
     opC.result().replaceAllUsesWith(gemm);
+    opC.erase();
```

Comment:
No need for this since follow on canonicalizations can clean it up.

Reply:
Yeah, could be. If one doesn't run the canonicalization pass, it results in incorrect output (I have checked). Erasing opC here should be fine? (opC isn't used anywhere else, so I believe it should always produce the correct code?)

Reply:
I think not erasing here violated the PXA requirement of having only one 'write' user, but regardless the erase certainly doesn't hurt, so I think leaving it is good.